sigma/pipelines/loki/loki.py

```diff
@@ -112,3 +113,86 @@ def loki_promtail_sysmon_message() -> ProcessingPipeline:
             ),
         ],
     )
+
+
+def loki_okta_system_log_json() -> ProcessingPipeline:
+    return ProcessingPipeline(
+        name="Loki Okta System Log json",
+        priority=20,
+        allowed_backends=frozenset({"loki"}),
+        items=[
+            ProcessingItem(
+                identifier="loki_okta_event_json_formatter",
+                transformation=SetCustomAttributeTransformation(
+                    attribute=LokiCustomAttributes.PARSER.value,
+                    value="json",
+                ),
+                rule_conditions=[
+                    LogsourceCondition(
+                        product="okta",
+                        service="okta",
+                    )
+                ],
+            ),
+            ProcessingItem(
+                identifier="loki_okta_field_name_mapping",
+                # Transform event fields names that should be camelCase
+                # See https://developer.okta.com/docs/reference/api/system-log/#logevent-object-annotated-example  # noqa: E501
+                transformation=FieldMappingTransformation(
+                    {
+                        v.lower(): v
+                        for v in [
+                            "eventType",
+                            "legacyEventType",
+                            "displayMessage",
+                            "actor_alternateId",
+                            "actor_displayName",
+                            "client_userAgent_rawUserAgent",
+                            "client_userAgent_os",
+                            "client_userAgent_browser",
+                            "client_geographicalContext_geolocation_lat",
+                            "client_geographicalContext_geolocation_lon",
+                            "client_geographicalContext_city",
+                            "client_geographicalContext_state",
+                            "client_geographicalContext_country",
+                            "client_geographicalContext_postalCode",
+                            "client_ipAddress",
+                            "debugContext_debugData_requestUri",
+                            "debugContext_debugData_originalPrincipal_id",
+                            "debugContext_debugData_originalPrincipal_type",
+                            "debugContext_debugData_originalPrincipal_alternateId",
+                            "debugContext_debugData_originalPrincipal_displayName",
+                            "authenticationContext_authenticationProvider",
+                            "authenticationContext_credentialProvider",

```

Comment:
Ah, nice catch! No, I missed it because it only appeared in some examples and not others :)

I deliberately excluded all maps that weren't explicitly defined and lists (especially given [the Loki `json` parser skips list](https://grafana.com/docs/loki/latest/logql/log_queries/#json)) - as otherwise I'd have to handle arbitrary properties/indexes - which would be a pain to handle, probably requiring a new `RegexFieldNameTransformation` or similar, and there are currently no example rules in the Sigma repository that use them.

Reply:
Yes, indeed, that's what I assumed when I saw [target](https://developer.okta.com/docs/reference/api/system-log/#target-object) (an array) was skipped. Makes sense! 😄